examples/playground/iframe.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
This is the document `iframe` startswith. Currently its set via the `src` attribute. Instead of serving this externally, we should see if we could insert it via the `srcdoc` attribute. I have not tried with this specific document. But I've tried the `srcdoc` attribute with another doc in #4279 and could not get it working.